src/components/teampage/team-member-styles.js

```diff
@@ -0,0 +1,189 @@
+import styled from "styled-components"
+import Img from "gatsby-image"
+import { devices } from "../../constants/devices"
+
+const fonts = {
```

Comment:
I don't think the typography stuff needs to be repeated, you could keep them in a `typography.js` file or somethign